e2etest/bgptests/bgp.go

```diff
@@ -662,6 +662,33 @@ var _ = ginkgo.Describe("BGP", func() {
 				return nil
 			}, 4*time.Minute, 1*time.Second).ShouldNot(HaveOccurred())
 
+			ginkgo.By("checking the sessions don't flap when changing the configuration")
+
+			previousNeighbors := map[string]frr.NeighborsMap{}
+			for _, c := range FRRContainers {
+				neighbors, err := frr.NeighborsInfoMap(c)
+				Expect(err).NotTo(HaveOccurred())
+				previousNeighbors[c.Name] = neighbors
+			}
+			ginkgo.By("creating another the service")
+			svc1, _ := testservice.CreateWithBackend(cs, testNamespace, "external-local-lb1", tweak)
+			testservice.Delete(cs, svc1)
+
+			Consistently(func() error {
+				for _, c := range FRRContainers {
+					neighbors, err := frr.NeighborsInfo(c)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(neighbors).To(HaveLen(len(previousNeighbors[c.Name])))
+
+					for _, n := range neighbors {
+						previousDropped := previousNeighbors[c.Name][n.IP.String()].ConnectionsDropped
+						if n.ConnectionsDropped > previousDropped {
+							return fmt.Errorf("increased connections dropped from %s to %s, previous: %d current %d", c.Name, n.IP.String(), previousDropped, n.ConnectionsDropped)
+						}
+					}
+				}
+				return nil
+			}, 5*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
```

Comment:
why 5 and not 10 or 20 seconds?

Reply:
I tested locally with a power machine and 5 seconds was still making the bug appear.
It's a `Consistently`, so the longer the time, the longer the execution time so we should be cautious when using them. If we starts seeing it flaking, we'll make it longer.

Reply:
hiding the error might be the case. I mean if you create the svc but process takes more than 5 seconds to apply the config, then your test will be green but behavior  can regress.

Reply:
yep, but the same can be said whatever time we put in. I can extend to 10, but I don't think it's worth doing more than that.

Reply:
we can let  5sec as long as you still think is good

Reply:
The test is failing on frr-k8s jobs, meaning that it is "at least" borderline to be good. Let's put 10 to be on the safe side. Once we see our tests are taking too long, we'll do a round to understand why.